content/content.go

```diff
@@ -65,6 +77,30 @@ func readFilesIntoFileContent(baseDir string, filelist []string) (map[string][]b
 	return filesContent, nil
 }
 
+// transformMetadataCondition takes content of condition field in metadata.yaml
+// and transforms it into a simple string
+func transformMetadataCondition(parsedCondition *interface{}, errorContent *RuleErrorKeyContent) error {
+	switch (*parsedCondition).(type) {
+	case string:
+		errorContent.Metadata.Condition = (*parsedCondition).(string)
+		return nil
+	case []interface{}:
```

Comment:
Does it work with `[]string` instead of `[]interface{}`? Probably if it is not an array of string we can stop parsing it

Reply:
Nope. There's no luck on that one, when unmarshalled It turns into a []interface and not directly into a []string